Articles/ShortArticles/PullRequestSizeConsiderations.md

```diff
@@ -1,4 +1,4 @@
-# PR Size Matters
+# PR Size Matters (Source)
```

Comment:
@bernhold, why is is this name-change needed?  If we need to edit the title in the source and the generated files to be different, that will complicate the process to generate files in the future.

Reply:
The front-end assumes that titles are unique.  If you have multiple items with the same title, *one* of them will be published and which one is not predictable.  This is true throughout the repo (i.e. not localized within a particular content-type directory, and also regardless of the publish metadata setting.

I agree this is not idea. If we're going to adopt the wikization approach as standard, we're going to have to deal with it.  But for now, we have to work around it.